viewer/packages/camera-manager/src/ComboControls.ts

```diff
@@ -77,7 +76,7 @@ export default class ComboControls extends EventDispatcher {
   public zoomToCursor = true;
   public minDeltaRatio = 1;
   public maxDeltaRatio = 8;
-  public minDeltaDownscaleCoefficient = 0.1;
+  public minDeltaDownscaleCoefficient = 0.05;
```

Comment:
Why change?

I don't think behaviour should change from this PR.

Reply:
Sure, will change to previous value. Was experimenting on this value for zooming from long distances. Will change it in a different PR, probably the one about controls sensitivity and small fixes for zooming speed.